test/kino/fs_test.exs

```diff
@@ -0,0 +1,68 @@
+defmodule Kino.FSTest do
+  use ExUnit.Case, async: true
+
+  describe "file_spec_to_fss/1" do
+    test "returns a file spec" do
+      path = "/home/bob/file.txt"
+      assert %FSS.Local.Entry{path: ^path} = Kino.FS.file_spec_to_fss(%{type: :local, path: path})
```

Comment:
Since this function should be private, we can test `Kino.FS.file_spec` instead. For this we need to fake communication with Livebook. You can do this:

```ex
test "returns a file spec" do
  name = "file.txt"
  path = "/home/bob/file.txt"
  spec = %{type: :local, path: path}

  configure_gl_with_reply({:livebook_get_file_entry_spec, name}, {:ok, spec})

  assert %FSS.Local.Entry{path: ^path} = Kino.FS.file_spec(name)
end

# ...

defp configure_gl_with_reply(request, reply) do
  gl =
    spawn(fn ->
      assert_receive {:io_request, from, reply_as, ^request}
      send(from, {:io_reply, reply_as, reply})
    end)

  Process.group_leader(self(), gl)
end
```

Reply:
Cool! Thanks! :D